lib/rules/selector-max-id/__tests__/index.mjs

```diff
@@ -524,3 +541,88 @@ testRule({
 		},
 	],
 });
+
+// with includeContextFunctionalPseudoClasses optional params
+
+testRule({
+	ruleName,
+	config: [
+		0,
+		{
+			includeContextFunctionalPseudoClasses: [':global'],
```

Comment:
```suggestion
			includeContextFunctionalPseudoClasses: [':--foo'],
```

Let's use custom pseudos in the test and documentation to avoid using non-standard syntax.

Reply:
Updated [f310fa7](https://github.com/stylelint/stylelint/pull/7380/commits/f310fa7a8a6ef9176c7587cb02b5bbad460ebd9f) 😄 thanks for your feedback!